schematic/store/synapse.py

```diff
@@ -1713,8 +1753,47 @@ def add_annotations_to_entities_files(
 
             # Adding annotations to connected files.
             if entityId:
-                self._add_annotations(dmge, row, entityId, hideBlanks, annotation_keys)
-                logger.info(f"Added annotations to entity: {entityId}")
+                # Format annotations for Synapse
+                annos_task = asyncio.create_task(
+                    self.format_row_annotations(
+                        dmge, row, entityId, hideBlanks, annotation_keys
+                    )
+                )
+                requests.add(annos_task)
+
+                while requests:
```

Comment:
This while loop is getting quite a bit nested, it might be a bit cleaner to extract this out to a function call. I am doing something similar here:
https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1107/files#diff-2dda10b4479134475ffeee7529eec430e40f828246ff532daa91b867d8d411b5R314
`async def _execute_download_tasks(self, download_tasks: Set[asyncio.Task]) -> None:`

Reply:
Thanks Bryan! I already addressed that :)